soc/st/stm32/stm32f4x/poweroff.c

```diff
@@ -11,17 +11,13 @@
 #include <zephyr/toolchain.h>
 #include <zephyr/drivers/misc/stm32_wkup_pins/stm32_wkup_pins.h>
 
-#include <stm32_ll_cortex.h>
+#include <stm32_common.h>
 #include <stm32_ll_pwr.h>
 
 void z_sys_poweroff(void)
 {
 	LL_PWR_ClearFlag_WU();
 
 	LL_PWR_SetPowerMode(LL_PWR_MODE_STANDBY);
-	LL_LPM_EnableDeepSleep();
-
-	k_cpu_idle();
-
-	CODE_UNREACHABLE;
+	stm32_enter_poweroff();
```

Comment:
Nitpicking: add an empty line above? (ditto in **soc/st/stm32/stm32wbx/poweroff.c**)